roles/openshift_version/tasks/main.yml

```diff
@@ -119,30 +127,42 @@
 
 - fail:
     msg: openshift_version role was unable to set openshift_version
```

Comment:
Suggestion: expand the messages in this section to suggest what to do (set openshift_{version,image_tag,pkg_version} in the inventory)?